packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -39,6 +42,43 @@ MeetingInfoUtil.getMeetingLink = (value) => {
   return sipUrl;
 };
 
+// modified from https://github.com/kirm/sip.js base
+// and with https://tools.ietf.org/html/rfc3261
+// requires the @ symbol
+MeetingInfoUtil.isSipUri = (sipString) => {
+  // eslint-disable-next-line no-warning-comments
+  // TODO: lets remove regex from this equation and user URI matchers and such
+  const sipRegex = /^(sips?)?:?(?:([^\s>:@]+)(?::([^\s@>]+))?@)([\w\-.]+)(?::(\d+))?((?:;[^\s=?>;]+(?:=[^\s?;]+)?)*)(?:\?(([^\s&=>]+=[^\s&=>]+)(&[^\s&=>]+=[^\s&=>]+)*))?$/;
+  const sipUri = sipRegex.exec(sipString);
+  return sipUri;
+};
+
+MeetingInfoUtil.generateOptions = (destination, type) => {
+  if (type) {
+    return {
+      destination,
+      type
+    };
+  }
+  const options = {};
+  if (MeetingInfoUtil.isSipUri(destination)) {
+    options.type = SIP_URI;
+    options.destination = destination;
+  }
+  else {
+    const convoUrl = url.parse(destination);
+    if (convoUrl && convoUrl.host && convoUrl.protocol) {
+      options.type = CONVERSATION_URL;
+      options.destination = destination;
+    }
+    else {
+      options.type = CONVERSATION_URL;
```

Comment:
We should change it to convo ID ? the type

Reply:
we generate a convo url, because that's all locus accepts to meeting info API